packages/connect/src/constants.ts

```diff
@@ -63,5 +61,37 @@ export function isValidPrefix(prefix: RelayPrefix): boolean {
   }
 }
 
-export const RELAY = `/${NAME}/relay/${version}`
-export const DELIVERY = `/${NAME}/delivery/${version}`
+export const MAX_RELAYS_PER_NODE = 5
+
+/**
+ * @param environment [optional] isolate from nodes running in other environments
+ * @returns the relay request protocol string
+ */
+export function CAN_RELAY_PROTCOL(environment?: string): string {
```

Comment:
new protocol: check if we can connect to relay and by doing this, check if relay is in the same environment